packages/grid/src/key-handlers/SelectionKeyHandler.ts

```diff
@@ -169,14 +174,22 @@ class SelectionKeyHandler extends KeyHandler {
           moveToRow,
           isShiftKey,
           true,
-          maximizePreviousRange
+          maximizePreviousRange,
+          stickyOptions
         );
       }
     } else {
       if (!grid.metrics) throw new Error('grid.metrics are not set');
 
       const { theme } = grid.props;
       const { autoSelectRow = false, autoSelectColumn = false } = theme;
+      // const stickyOptions: StickyOptions = {
+      //   shouldStickBottom:
+      //     event.key === 'ArrowDown' ||
+      //     event.key === 'End' ||
+      //     event.key === 'PageDown',
+      //   shouldStickRight: event.key === 'ArrowRight',
+      // };
```

Comment:
Still need to delete this unused code

Reply:
Ah sorry about that I accidentally made that change on the e2e context menu branch rather than this one. Should be good to go now